extensions/openapi-generator/deployment/src/main/resources/templates/api.qute

```diff
@@ -0,0 +1,56 @@
+package {package};
+
+{#for imp in imports}
+import {imp.import};
+{/for}
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.List;
+import java.util.Map;
+import javax.ws.rs.*;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.MediaType;
+import org.jboss.resteasy.annotations.jaxrs.*;
+
+{#insert disableMultipart}
+import org.apache.cxf.jaxrs.ext.multipart.*;
```

Comment:
This is only for RESTEasy Classic, I think? @michalszynkiewicz we support multipart from the RESTEasy Reactive client, right?

Reply:
yes, https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/rest-client-reactive.adoc#multipart-form-support